docs/quantize/quantize_example.py

```diff
@@ -0,0 +1,104 @@
+from functools import partial
```

Comment:
Umm, Sorry, This file may should put under the  `blueoil/networks/` directory.
And it's good to separate to 2 PRs. 

1. Example code PR.
2. Documentation PR.

Reply:
1. will be in #966